_data/simple-icons.json

```diff
@@ -4785,7 +4789,7 @@
         {
             "title": "Kaufland",
             "hex": "E10915",
-            "source": "https://www.kaufland.com/etc.clientlibs/kaufland/clientlibs/clientlib-klsite/resources/frontend/img/kl-logo-small-e825b661c5.svg"
+            "source": "https://commons.wikimedia.org/wiki/File:Kaufland_201x_logo.svg"
         },
```

Comment:
A SVG is available in their website footer. It would be better to have https://www.kaufland.com as source